basics/signals/src/panel.ts

```diff
@@ -3,24 +3,23 @@ import {
   nullTranslator,
   TranslationBundle
 } from '@jupyterlab/translation';
-import { StackedPanel } from '@lumino/widgets';
+
+import { Panel } from '@lumino/widgets';
 
 import { ButtonWidget, ICount } from './button';
-/**
- * The class name added to console panels.
- */
-const PANEL_CLASS = 'jp-tutorial-view';
```

Comment:
Could you keep the class name as constant as this is to align with JupyterLab practice?

Reply:
Fixed as requested.  However, I noticed that the documentation seemed to indicated that they were moving away from that practice: "CSS classnames are defined inline in the code. We used to put them as all caps file-level consts, but we are moving away from that." (https://jupyterlab.readthedocs.io/en/stable/developer/css.html)

Reply:
Thanks for pointing this out - I miss that information. I have opened a new issue to conform to that principle. So we can merge this one.